crates/storage/provider/src/providers/blockchain_provider.rs

```diff
@@ -1502,20 +1505,38 @@ mod tests {
         chain_spec: Arc<ChainSpec>,
         database_blocks: usize,
         in_memory_blocks: usize,
+        tx_count: usize,
```

Comment:
you can use the field from the `BlockRangeParams` struct

Reply:
Just fixed, used the end of the range each time, let me know what you think, seems good now, no extra field